pyproject.toml

```diff
@@ -1,20 +1,20 @@
 [tool.poetry]
```

Comment:
Why did so much of this file change? Not that it seems wrong, just doesn't seem like the issue to do this in.

Reply:
Auto-sorting...I noticed you asked to re-alphabetise, I didn't realise the autoformatter did so much! Will revert

Reply:
Sorry, I meant just the dependency, heh